specification/cognitiveservices/data-plane/Speech/VideoTranslation/preview/2024-05-20-preview/video-translation.json

```diff
@@ -0,0 +1,963 @@
+          "$ref": "#/definitions/ResourceId"
+        },
+        "status": {
+          "$ref": "#/definitions/Status",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceId": {
+      "description": "Resource id",
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9][a-zA-Z0-9._-]{1,62}[a-zA-Z0-9]$",
+      "minLength": 3,
+      "maxLength": 64
+    },
+    "Status": {
+      "description": "Status of a resource.",
+      "enum": [
+        "NotStarted",
+        "Running",
+        "Succeeded",
+        "Failed"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "Status",
+        "modelAsString": true
+      }
+    },
+    "TranslationCreationWebvttFileFormat": {
+      "description": "Translation creation file kind.",
+      "enum": [
+        "SourceLocaleSubtitle",
+        "TargetLocaleSubtitle",
+        "MetadataJson"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "TranslationCreationWebvttFileFormat",
+        "modelAsString": true
+      }
+    },
+    "VideoTranslationVoiceKind": {
+      "description": "Translation voice kind.",
+      "enum": [
+        "PlatformVoice",
+        "PersonalVoice"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "VideoTranslationVoiceKind",
+        "modelAsString": true
+      }
+    },
+    "PaginatedTranslations": {
+      "description": "Paginated translation list",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Translation list",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Translation"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page.",
+          "format": "uri",
+          "type": "string"
+        }
+      }
+    },
+    "PaginatedIterations": {
+      "description": "Paginated iteration list",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Iteration list",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Iteration"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page.",
+          "format": "uri",
+          "type": "string"
+        }
+      }
+    },
+    "WebvttFile": {
+      "description": "Translation webvtt file input.",
+      "required": [
+        "webvttFileUrl",
+        "webvttFileFormat"
+      ],
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "Translation webvtt file url",
+          "type": "string",
+          "format": "uri"
+        },
+        "format": {
+          "$ref": "#/definitions/TranslationCreationWebvttFileFormat"
+        }
+      }
+    },
+    "Translation": {
+      "description": "Translation response.",
+      "required": [
+        "id",
+        "sourceLocale",
```

Comment:
These moved to input. It's strange that swagger editor didn't report grammar error.